packages/client/lib/Nft.ts

```diff
@@ -0,0 +1,60 @@
+import { NftProvider, Address, Transaction } from '@liquality/types'
+
+export default class Nft implements NftProvider {
+  client: any
+
+  constructor(client: any) {
+    this.client = client
+  }
+
+  /** @inheritdoc */
+  async balance(contract: Address | string, tokenIDs?: number | number[]): Promise<number | number[]> {
+    const balance = await this.client.getMethod('balance')(contract, tokenIDs)
+    return balance
+  }
+
+  /** @inheritdoc */
+  async transfer(
+    contract: Address | string,
+    receiver: Address | string,
+    tokenIDs: number | number[],
```

Comment:
Do most nft standards have ability to transfer multiple in 1 call? If not, best to have to separate functions. `transfer` and `transferBatch`

This way, a provider could implement either, or throw an error if that function is not possible